kratos/geometries/triangle_3d_3.h

```diff
@@ -838,7 +838,7 @@ template<class TPointType> class Triangle3D3
         const double Tolerance = std::numeric_limits<double>::epsilon()
         ) override
     {
-        PointLocalCoordinates( rResult, rPoint );
+        PrivatePointLocalCoordinates( rResult, rPoint, true );

```

Comment:
A minor comment, the standard way to say that a method is internal is to name it with Implemenation and make it private method: 
```c++
private:
PointLocalCoordinatesImplementation(){ ...}
```

Reply:
I used that name because we chose that name for the Quad. Do I change both?

Reply:
It would be better to change them so.

Reply:
Okidoki